src/main/java/com/wynntils/modules/chat/configs/ChatConfig.java

```diff
@@ -40,13 +40,19 @@
     @Setting(displayName = "Filter Info Messages", description = "Should Wynncraft info messages be hidden from chat?\n\n§8Messages starting with §4[Info]§8 will no longer appear in chat.", order = 8)
     public boolean filterWynncraftInfo = true;
 
-    @Setting(displayName = "Filter Join Messagse", description = "Should Wynncraft join messages be hidden from chat?", order = 9)
+    @Setting(displayName = "Filter Join Messages", description = "Should Wynncraft join messages be hidden from chat?", order = 9)
     public boolean filterJoinMessages = false;
 
-    @Setting(displayName = "Filter Territory Enter", description = "Should territory enter messages be hidden from chat?\n\n§8Territory enter messages look like §7[You are now entering Detlas]§8.", order = 10)
+    @Setting(displayName = "Filter Event Messages", description = "Should Wynncraft join messages be hidden from chat?\n\n§8Messages starting with §6[Event] will no longer appear in chat.", order = 10)
+    public boolean filterEventMessages = false;
+
+    @Setting(displayName = "Filter Territory Enter", description = "Should territory enter messages be hidden from chat?\n\n§8Territory enter messages look like §7[You are now entering Detlas]§8.", order = 11)
     public boolean filterTerritoryEnter = true;
 
-    @Setting(displayName = "Show Held Item Chat Message", description = "Should details of your compass and soul points be shown in chat while you are holding them?", order = 12)
+    @Setting(displayName = "Filter Party Finder Messages", description = "Should Party Finder recommendation messages be hidden from chat?.", order = 12)
+    public boolean filterPartyFinder = true;
```

Comment:
Default false, party finder messages are useful for new players and increase quite a lot the use of the party finder, they just need to be disabled if the person is absolutely sure they don't want to see it anymore!
```suggestion
    public boolean filterPartyFinder = false;
```